provenance_store/provenance_store/settings.py

```diff
@@ -76,8 +77,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': os.getenv("POSTGRES_DB"),
+        'USER': os.getenv("POSTGRES_USER"),
+        'PASSWORD': os.getenv("POSTGRES_PASSWORD"),
+        'HOST': os.getenv("POSTGRES_SERVER"),
```

Comment:
At least this one should probably default to the service name from the compose file.

Reply:
and the var name oughta be `POSTGRES_HOST` rather than `_SERVER`